count_chars.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Source: https://github.com/cmroughan/kraken_generated-data/blob/master/tools/count_chars.py
```

Comment:
Note that this is GPL code. Not sure about the consequences.

Reply:
Using GPL code requires changing the license for tesstrain to GPL. I don't think that we should do this, so GPL code cannot be used here.

Reply:
@cmroughan, are you the author of that code, and would you agree to use it here under the Apache 2.0 License?

Reply:
I am, and I wouldn't mind its use here under Apache 2.0. I will follow up with a pull request.